.github/workflows/drift.yml

```diff
@@ -37,6 +37,10 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
+      - name: Install libsqlite3
+        if: matrix.target == 'linux'
+        run: sudo apt-get -y install libsqlite3-dev
+
```

Comment:
I think we should do this in the shared action even though it may not be necessary on all flutter builds: https://github.com/getsentry/sentry-dart/blob/0c0805414333c4e379a41ab179dd5749b49dec27/.github/actions/flutter-test/action.yml#L29

Reply:
yea we can do that, I'll move it in another PR